sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -126,14 +179,20 @@ def train(
         from kubeflow.storage_initializer.s3 import S3DatasetParams
         from kubeflow.storage_initializer.hugging_face import (
             HuggingFaceModelParams,
-            HfDatasetParams,
+            HuggingFaceDatasetParams,
+        )
+
+        print(
+            "Thank you for using `train` API for LLMs fine-tuning. This feature is in alpha stage "
```

Comment:
What is the definition of the `alpha stage` in SDK?
If we say that this feature is still alpha stage, defininig criteria to graduate them to beta and GA would be better.

Reply:
Yes, I know that @StefanoFioravanzo is working on this as part of this effort: https://github.com/kubeflow/community/issues/709. When we have some time we can discuss what is the graduation steps for our new features.

Also, we do have criteria for Kubeflow components here: https://www.kubeflow.org/docs/started/support/#application-status, but it is not related to the components features.

Reply:
But, both (https://github.com/kubeflow/community/issues/709 and https://www.kubeflow.org/docs/started/support/#application-status) seem to be for components, not SDK.

This PR seems to introduce a dedicated graduation cycle for SDK separated from the application above, right?

Reply:
@tenzen-y If you check @StefanoFioravanzo 's issue he also says this:
>With "component" I not only mean an entire project/repo but also significant features within those projects (think about the new Fine Tune APIs for LLM).

So we also consider on how we should define graduation criteria for our new big features.

Reply:
Oh, I have missed that sentence. Thank you!